android/filament-android/libfilament-jni.map

```diff
@@ -1,4 +1,20 @@
 LIBFILAMENT {
-  global: *filament*; JNI*;
+  global:
+    Java_com_google_android_filament_*;
+    JNI*;
+    *filament*Camera*;
```

Comment:
Just curious, is `*filament*` not sufficient here?

Reply:
It should be but it will export more than what we need.

Reply:
It is sufficient but it is too wide of a net, e.g. it exposes UibGenerator.